docs/src/tangent-models.md

```diff
@@ -0,0 +1,34 @@
+# Tangent space models
+
+One of the more popular methods of statistical modeling on manifolds is selecting a point $p$ on the manifold $\mathcal M$ (for example Riemannian center of mass of data points), transforming all point to the tangent space $T_p\mathcal M$ at that point, and using traditional linear models in that tangent space.
+Tangent space Principal Component Analysis is a popular example of this kind of modeling.
+The example below demonstrates how to build and use such model.
+
+```@example
+using Manifolds, MultivariateStats
+M = Sphere(2)
+pts = [project(M, randn(3)) for _ in 1:100]
+m = mean(M, pts)
+logs = log.(Ref(M), Ref(m), pts)
+basis = DefaultOrthonormalBasis()
+coords = map(X -> get_coordinates(M, m, X, basis), logs)
+coords_red = reduce(hcat, coords)
+z = zeros(manifold_dimension(M)))
+model = fit(PCA, coords_red; maxoutdim=1, mean=z)
+X = get_vector(M, m, reconstruct(model, [1.0]), basis)
+geodesic(M, m, X, range(-1.0, 1.0, length=101))
```

Comment:
This last line is just for visualization, I think `X should be the result`.

Reply:
OK, I'll change that.